front/app/components/Form/Components/Controls/TopicsControl.tsx

```diff
@@ -22,14 +23,19 @@ const TopicsControl = ({
   id,
   required,
 }: ControlProps) => {
-  const availableTopics = uischema?.options?.available_topics ?? [];
+  const availableTopics =
```

Comment:
Moved topics to the oneOf option instead of this custom available opic to fit the JSONSchema standard better.